l2geth/rollup/sync_service.go

```diff
@@ -435,10 +435,10 @@ func (s *SyncService) SequencerLoop() {
 			log.Error("Could not sequence", "error", err)
 		}
 		s.txLock.Unlock()
+	}
 
-		if err := s.updateContext(); err != nil {
-			log.Error("Could not update execution context", "error", err)
-		}
+	if err := s.updateContext(); err != nil {
+		log.Error("Could not update execution context", "error", err)
 	}
 }
 
```

Comment:
Just leaving myself a mental note that we'll need to update the documentation to reflect this new behavior.